lib/dialogue-agent/entity-linking/entity-finder.js

```diff
@@ -64,6 +65,7 @@ function getBestEntityMatch(searchTerm, entityType, candidates) {
         score -= 0.1 * editDistance(searchTerm, candDisplay);
 
         let candTokens = candDisplay.split(' ');
+        candTokens = new Set(candTokens);
```

Comment:
I made this change because previously if I tried to look for the album "the wall", the getBestEntityMatch function would return "The Writing's On The Wall" as the proper match instead of "The Wall" because the keyword "the" was matched twice on  "The Writing's On The Wall"

Reply:
"The Writing's On The Wall" has a pretty high edit distance compared to "The Wall" because of the number of deleted characters. I'm surprised by this change. Maybe we just need to lower the term for "the"?

Reply:
Anyway, can you add tests in test/unit/test_entities for this?